src/app/App.jsx

```diff
@@ -1,55 +1,43 @@
-import React, { Component } from "react";
+import React, { Component, useEffect, useState } from "react";
```

Comment:
Am I right in thinking since this file is moving from a class component to a functional component we can stop importing Component?

Reply:
sure!